interpreter/core/core.py

```diff
@@ -50,15 +53,21 @@ def __init__(self):
         self.gguf_quality = None
 
         # Load config defaults
-        config = get_config()
-        self.__dict__.update(config)
+        self.extend_config(self.config_file)
 
         # Check for update
         if not self.local:
             # This should actually be pushed into the utility
             if check_for_update():
                 display_markdown_message("> **A new version of Open Interpreter is available.**\n>Please run: `pip install --upgrade open-interpreter`\n\n---")
 
+    def extend_config(self, config_path):
```

Comment:
We could also call this `load_config` instead of `extend_config` and then it would be more clear what's going on when you call `interpreter.load_config()` via Python.